app/Repositories/Eloquents/PostRepository.php

```diff
@@ -158,4 +159,41 @@ public function search($query, $input, $limit, $isPaginate = false)
 
         return $query;
     }
+
+    public function increaseViewTotal($id)
+    {
+        $this->model()->find($id)->increment('total_view', 1);
+    }
+
+    public function findPostWithUser($id)
+    {
+        return $this->model()->with('user')->with('tags')->findOrFail($id);
+    }
+
+    // get replies with posts
+    public function getRepliesFromPost($post)
+    {
+        return $post->replies()->with('user')->get();
+    }
+
+    // get replies with posts
+    public function findVoteFromPost($post)
+    {
+        return $post->votes()->where(['voteable_id' => $post->id, 'user_id' => \Auth::id()])->first();
+    }
+
+    public function getRelatedPost($listTag, $post_id)
+    {
+        $relatedPosts = collect([]);
+        foreach ($listTag->pluck('id') as $key => $tag) {
+            $relatedPosts = $relatedPosts->merge(Tag::with('posts')->find($tag)->posts);
+        }
+
+        $relatedPosts = $relatedPosts->unique('id');
+        $relatedPosts = $relatedPosts->filter(function($item) use ($post_id) {
+            return $item->id != $post_id;
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/PostRepository.php</br>**Line**: 194</br>Variable "post_id" is not in valid camel caps format